nat.red

```diff
@@ -0,0 +1,13 @@
+import path
+
+let pred (x : nat) : nat =
+  nat-rec x with
+  | zero ⇒ zero
+  | suc n ⇒ n
+  end
+
+let pred/succ/zero (x : nat) : Path nat x (pred (suc x)) =
```

Comment:
By the way, it's worth observing that this holds by *case*, without needing induction. This is because in each case, the equation holds definitionally (but not prior to the case split).